Optimize (CQA) - Mng content - Synchronizing (#4872)
theforeman/foreman-documentation: guides/common/modules/proc_synchronizing-repositories-by-using-web-ui.adoc

= Synchronizing repositories by using {ProjectWebUI}

[role="_abstract"]

> Run an ad hoc synchronization for selected repositories to download the latest upstream content before publishing it to hosts.

Reviewer: I kinda prefer the version in CLI better because there are other steps involved before hosts consume it:

```suggestion
Run an ad hoc synchronization for selected repositories to download the latest upstream content into {Project}.
```
